scipy/integrate/_ivp/common.py

```diff
@@ -118,7 +127,10 @@ def select_initial_step(fun, t0, y0, f0, direction, order, rtol, atol):
     else:
         h1 = (0.01 / max(d1, d2)) ** (1 / (order + 1))
 
-    return min(100 * h0, h1)
+    h = min(100 * h0, h1)
```

Comment:
Python supports `min` with many arguments, I'd condense everything in one line. I think it will be quite readable and easy to understand. 

Also might denote `abs(t_bound - to)` as some parameter as it's used 2 times.